src/chrome/content/rules/Vancity.xml

```diff
@@ -0,0 +1,38 @@
+<!--
+	Invalid certificates:
+		- annualreport.vancity.com
+		- bcp.vancity.com
+		- blog.vancity.com
+		- holidays.vancity.com
+		- images.vancity.com
+		- weallprofit.vancity.com
+		- vancityinsurance.com (www. has a valid certificate)
+		- myreloadable.ca (www. has a valid certificate)
+-->
+
+<ruleset name="Vancity">
+	<target host="vancity.com" />
+	<target host="www.vancity.com" />
+	<target host="support.vancity.com" />
+	<target host="onlinebusinessplus.vancity.com" />
+	<target host="mdws.vancity.com" />
+	<target host="adfs1.vancity.com" />
+	<target host="moveit.vancity.com" />
+	<target host="download.vancity.com" />
+	<target host="webmail.vancity.com" />
+	<target host="myvisaaccount.com" />
+	<target host="www.myvisaaccount.com" />
+	<target host="vancityinsurance.com" />
+	<target host="www.vancityinsurance.com" />
+	<target host="myreloadable.ca" />
+	<target host="www.myreloadable.ca" />
+
+	<test url="https://vancityinsurance.com/" />
+	<test url="https://myreloadable.ca/" />
+
+	<securecookie host=".+" name=".+" />
+
+	<rule from="^https?://vancityinsurance\.com/" to="https://www.vancityinsurance.com/" />
+	<rule from="^https?://myreloadable\.ca/" to="https://www.myreloadable.ca/" />
```

Comment:
I initially added those rules because `vancityinsurance.com` and `myreloadable.ca` both have valid certs only for `www.`. Since we can't do an https-to-https redirect, I'm removing the rules and targets.